### ISSUE TITLE
Trouble with Movecraft compatibility with Bukkit jars

### ISSUE THREAD

**Issue by MRGhorm1379** [msummers123/Movecraft-3#80]
I've recently created a Bukkit Minecraft server and would like to implement Movecraft into it, however I'm having trouble matching a bukkit server jar and a Movecraft jar so that it works. For some reason even if the versions say they match, the plugin has issues. Currently we're using Bukkit 1.7.9-R0.2 with MC version 1.7.10. I'm using the corresponding plugin for movecraft, but it still doesn't work like it should.
The plugin seems to load just fine, but whenever I try to click on a sign, I get the following error: "CONSOLE: ERROR]: Could not pass event PlayerInteractEvent to Movecraft v3.1.X60
 CONSOLE: Source) ~[?:?]"
We're using versions of Bukkit which are compatible with McMyAdmin (which I don't think are much different if not different at all), however they're corresponding versions from a while ago. We were running an older server, and got these versions from there. They seem to appropriate builds and whatnot, the point is though that they were not recently downloaded from Bukkit, because what versions we tried to download from Bukkit wouldn't work.

Not exactly sure what to do, where to get my downloads from, and which Bukkit versions go with what Movecraft versions. Any help with clarification would be much appreciated. 

Thanks!

*edit:
After seeing what information I'm supposed to provide, here are the specific Movecraft and Bukkit builds I tried: (using the /version command on the server) Bukkit-1.7.9-R0.2-16-g37c7969-b3105jnks (MC: 1.7.10) or 1.7.0_55 if that's any easier to understand. 
For Movecraft: V 3.1.X54 for 1.7.10

**Comment by msummers123** [comment]
Well, firstly you should actually use the latest version of the plugin regardless of the stated version number. These days it actually auto-detects the correct core map updater to use, so the newest version is pretty much always better.

But that's not the problem you are having. I will need to see more of the error to tell for sure, but it sounds like an error that can occur if it can not find any type files.

Did you extract the types you want to use on your server and place them in the types directory as described in the tutorial and the quick start guide?

**Comment by MRGhorm1379** [comment]
Ah, That's actually a really neat plugin feature!

Whups, I'm sorry, I did not. I did however look through the startup console for any Movecraft errors, but I couldn't seem to find any, and particularly did not see an error stating that I didn't have any craft files. I probably just accidentally looked it over. I downloaded the example .craft files, installed them, and it started working just fine.

Thanks for your help! Sorry I skipped over the part of the installation process which I thought I knew, but apparently forgot. By the way, Compliments on your plugin, it's most definitely a really cool and fun one!
